fxgl-core/src/main/kotlin/com/almasb/fxgl/core/EngineService.kt

```diff
@@ -52,6 +52,16 @@ abstract class EngineService : Updatable, SerializableType {
      */
     open fun onExit() { }
 
+    /**
+     * Called just before the main loop is paused, which is only happens when the main window is minimized.
```

Comment:
> which is only happens when the main window is minimized.

I think we should remove this bit, because in the future it may not be true. Plus, it refers to main window being minimized, but on mobile it would be called when the app enters a paused state. So just leave:

`Called just before the main loop is paused.`